### PR TITLE
get notebook css from cdn.jupyter.org

### PR DESCRIPTION
**Pull request by minrk**
removes dependency on the notebook package

This will only be 4.0.0-dev while notebook is unreleased, after which point it will be pinned to a stable release, even for dev versions of nbconvert.

### CODE REVIEW

**Comment by rgbkrk** [comment]
A whole CDN for one file.

Excited to see a green light here.

**Comment by rgbkrk** [comment]
Welp.

``` 
    Failed to download css from https://cdn.jupyter.org/notebook/4.0.0-dev/style/style.min.css: addinfourl instance has no attribute '__exit__'
    
    error: Need Notebook CSS to proceed: nbconvert/resources/style.min.css
```

**Comment by minrk** [comment]
Oops. That's me forgetting about Python 2. urlopen isn't a context manager on py2.

**Comment by damianavila** [comment]
There is still an error but I am :+1: on the idea...

**Comment by minrk** [comment]
After a bit more fiddling, Builds passing now.

File: nbconvert/preprocessors/csshtmlheader.py [hunk 1]
```diff
@@ -67,10 +69,15 @@ def _generate_header(self, resources):
         # Load the user's custom CSS and IPython's default custom CSS.  If they
         # differ, assume the user has made modifications to his/her custom CSS
         # and that we should inline it in the nbconvert output.
+        try:
+            from notebook import DEFAULT_STATIC_FILES_PATH
+        except ImportError:
+            DEFAULT_STATIC_FILES_PATH = None
```
Comment (rgbkrk) [inline nbconvert/preprocessors/csshtmlheader.py]: Why keep the fallback if notebook is no longer a dependency?

**Pull request merged**